4-distributed-scheduler.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "9d19f489-59e6-48c7-aade-25530dcf253c",
```

Comment:
<p>Since this notebook is intended to precede the debugging performance notebook, should probably be good to remove the additional resources here (and move it to whatever the final notebook in the tutorial is)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jsignell/dask-tutorial-advanced/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='92'/>